azure-pipelines.yml

```diff
@@ -51,7 +51,7 @@ jobs:
       export PATH=/usr/local/opt/flex/bin:/usr/local/opt/bison/bin:$PATH;
       mkdir -p $(Build.Repository.LocalPath)/build
       cd $(Build.Repository.LocalPath)/build
-      cmake .. -DPYTHON_EXECUTABLE=$(which python3) -DCMAKE_INSTALL_PREFIX=$HOME/nmodl -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      cmake .. -DPYTHON_EXECUTABLE=$(which python3) -DCMAKE_INSTALL_PREFIX=$HOME/nmodl -Dnmodl_PYTHONPATH=$HOME/nmodl -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

Comment:
why `nmodl_PYTHONPATH` does not have nmodl with capital letters?

Reply:
This comes from [here](https://github.com/BlueBrain/CoreNeuron/blob/6da0f49114c79d26b422ff897ed936af235e4811/CMake/packages/Findnmodl.cmake#L36) and this is something we have to fix in `CoreNEURON` if the path has changed I believe